grpc/tests/grpctest.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import print_function
+
+import os
+import sys
+import grpc
+import flatbuffers
+
+from concurrent import futures
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'tests'))
+import MyGame.Example.Monster as Monster
+import MyGame.Example.Stat as Stat
+import MyGame.Example.monster_test_grpc_fb as monster_grpc_fb
+
+test_stat_id = "test_stat_id"
+test_monster_name = "test_monster_name"
+
+
+class MonsterStorage(monster_grpc_fb.MonsterStorageServicer):
+
+    def Store(self, request, context):
+
+        m = Monster.Monster().GetRootAsMonster(request, 0)
+
+        b = flatbuffers.Builder(0)
+        i = b.CreateString(test_stat_id)
+        Stat.StatStart(b)
+        Stat.StatAddId(b, i)
+        b.Finish(Stat.StatEnd(b))
+        return bytes(b.Output())
```

Comment:
I was following the previous PR. Note that `bytes()` makes a copy of the bytearray https://stackoverflow.com/questions/35880065/does-converting-from-bytearray-to-bytes-incur-a-copy
In order to avoid this, it would need a change on GRPC side which only allow bytes type (not bytearray) :/